packages/components/src/item-group/item/hook.ts

```diff
@@ -13,26 +13,47 @@ import { useItemGroupContext } from '../context';
 import { useCx } from '../../utils/hooks/use-cx';
 import type { ItemProps } from '../types';
 
+function useDeprecatedProps( {
```

Comment:
Not sure we need this since this component is experimental and probably only used in Gutenberg.

Reply:
`ItemGroup` was promoted in #33701 and released in Gutenberg 11.3, but it's always been marked as experimental and, as you said, doesn't probably any usages outside of Gutenberg. I will go ahead and remove all traces of `isAction`

Reply:
Done in [`a148cea` (#35152)](https://github.com/WordPress/gutenberg/pull/35152/commits/a148ceaa6384b1ee34d92512c369aa5e897dc7fa)